map180/map.go

```diff
@@ -408,13 +408,13 @@ func (m *map3857) nePolySVG(zoom int, layer int) (string, error) {
 	switch m.crossesCentral {
 	case true:
 		//  things to the left of 180.
-		db.QueryRow(svgPolyQuery,
+		_ = db.QueryRow(svgPolyQuery,
```

Comment:
About the ignoring of this error, refer to the comment in the beginning of func.